x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -133,6 +134,14 @@ deprecated_metadata
     : OPENING_BRACKET metadataOption CLOSING_BRACKET
     ;
 
+metricsCommand
+    : METRICS fromIdentifier (COMMA fromIdentifier)* metadataOption? fromOptions? statsOption?
+    ;
+
+statsOption
+    : STATS aggregates=fields? (BY grouping=fields)?
```

Comment:
Compared to our document, I've added a `STATS` keyword between the indices and the aggregations. This clarifies the distinction between options and aggregations within the METRICS command. I'm find to remove it if needed. `METRICS pods OPTIONS "allow_no_indices"="true" STATS load=avg(cpu) BY pod | SORT pod`

Reply:
Yeah I'm on the fence here. Do we have documentation on how OPTIONS is used in general?

Also, can we get an example with filtering through WHERE? Or, do you plan to include this in a follow-up?

Reply:
This is the index options for ES|QL: https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-index-options.html

Reply:
++ I have pushed https://github.com/elastic/elasticsearch/pull/108115/commits/739e00af700dd685f9656b8ce44f04a260f12bfe to implement the original proposal.